cloud-service-providers/azure/aks/README.md

```diff
@@ -0,0 +1,17 @@
+# nim on aks
+
+AKS need to be properly created in order to be able deploy NIM on it.  There are two things need to be taken care of.  Make sure you have right GPU for NIM and right driver version as well.  Unfortunately, default GPU driver in AKS usually always a little bit too old for latest NVidia software in general.  Microsoft has no official solution for it yet.  There is a preview version of CLI to solve this issue.  We need to create AKS using preview version of CLI.  Prerequisite section talks about how to set up your local environment to enable you to create AKS from preview version of CLI.
```

Comment:
Can we break this into two paragraphs and word it more passively. Something lik:

To deploy NIM on AKS successfully, ensure you have the right GPU and driver version. The default GPU driver in AKS is usually outdated for the latest NVIDIA software, and Microsoft does not yet have an official solution for this issue.

To resolve this, use the preview version of the CLI to create the AKS cluster. The Prerequisites section explains how to set up your local environment to enable AKS creation with the preview CLI.

Reply:
done